packages/camera/lib/new/src/camera_controller.dart

```diff
@@ -59,6 +65,9 @@ class CameraController {
     );
   }
 
+  // Keep only one active instance of CameraController.
+  static CameraController _instance;
```

Comment:
May I offer a different view about this approach? How about not having it as a static field? If the client developer unwisely calls `initialize` without disposing the previous CameraController, this may be a error, but also that may be that he is trying to access more than one camera at the same time (I don't know if this is possible in the hardware layer), but seems feasible to have more than one camera open and this to be alright.

Even if there is no chance of the developer accessing more than one camera at the same time, I believe that an exception should be raised explaining that the device is busy.

Reply:
Thanks for the review!

Our reasoning to keep this functionality is because the current implementation of the plugin only allows one controller to be active at a time. It isn't documented in the plugin, but that is how it works underneath. While we refactor the platform specific code, we want to keep the interface the same and not cause many breaking changes.

The other reason for this functionality is because the `CameraController` will only serve as a simple to use and narrowly defined interface usable on any platform. For more complex features (like accessing multiple cameras), we will also expose access to this through another interface or require platform specific code.